hooks/useDestroyer.ts

```diff
@@ -137,13 +146,33 @@ export function useDestroyer() {
   //   [commitDestroyFile, handleResponse]
   // );
 
+  /* Disable or destroy an ordering */
+  const [
+    commitDisableOrDestroyOrdering,
+  ] = useMutation<useDestroyerDestroyOrderingMutation>(destroyOrderingMutation);
+
+  const ordering = useCallback(
+    async (input: DestroyOrderingInput, label: string) => {
+      const response = await commitDisableOrDestroyOrdering({
+        variables: { input },
+      });
+      return handleResponse(
+        response.destroyOrdering,
+        label,
+        response.destroyOrdering?.disabled
+      );
```

Comment:
@jen-castiron I'm struggling with the types in `handleResponse` so that I don't have to pass `disabled` in separately like this, since `disabled` doesn't exist on `useDestroyerFragment`. What's a better way to handle this? I'm also happy for any other feedback you have on this when you get a chance. Thanks!

Reply:
I spent some time working on the types and came up with something I think is better—see comment below.